arlo_server/routes.py

```diff
@@ -1416,9 +1434,10 @@ def clear_loggedin_user():
 def serialize_election(election):
     return {
         "id": election.id,
-        "name": election.name,
+        "auditName": election.audit_name,
+        "electionName": election.election_name,
         "state": election.state,
-        "election_date": election.election_date,
+        "electionDate": election.election_date,
```

Comment:
```suggestion
        "electionDate": isoformat(election.election_date),
```

If we do this, we should change the signature of `isoformat` to this:

```py
def isoformat(datetime: Optional[Union[datetime.datetime, datetime.date]]) -> Optional[str]:
```

Reply:
Will do